src/applications/vaos/tests/new-appointment/components/DateTimeRequestPage.unit.spec.jsx

```diff
@@ -197,7 +197,7 @@ describe('VAOS <DateTimeRequestPage>', () => {
       userEvent.click(buttons[0]);
 
       // 2. Simulate user selecting AM
-      let checkbox = await screen.findByRole('checkbox', {
+      let checkbox = await screen.getByRole('checkbox', {
```

Comment:
These need to be `findByRole` rather than `getByRole`, since they're using `await`. Was this to fix an error?